internal/juju/secrets.go

```diff
@@ -170,27 +170,21 @@ func (c *secretsClient) UpdateSecret(input *UpdateSecretInput) error {
 		value = map[string]string{}
 	}
 
-	if input.SecretId == "" {
-		// Specify by name
-		err = secretAPIClient.UpdateSecret(nil, *input.Name, input.AutoPrune, "", description, value)
-		if err != nil {
-			return typedError(err)
-		}
-	} else {
+	if input.SecretId != "" {
```

Comment:
Suggest
```
If input.SecretID == "" {
   return fmt.Error()
}
```
Allows for less indentation in the code and easier reading. Also ensures we error rather than pretending the update was successful.

Reply:
done